crates/extensions/tedge_mqtt_ext/Cargo.toml

```diff
@@ -9,8 +9,13 @@ license = { workspace = true }
 homepage = { workspace = true }
 repository = { workspace = true }
 
+[features]
+# No features on by default
+default = []
+test-helpers = ["assert-json-diff"]
```

Comment:
I think it's recommended to use `dep:assert-json-diff` rather than the implicit feature name these days (https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies).